primer.html

```diff
@@ -307,6 +307,8 @@ <h2>ShapeTree Structure <a class="cross" href="spec#structure"></a></h2>
         </li>
         <li>an optional <dfn>contents</dfn>: a list of nested steps.<br/>
         This captures the containership of a conforming resource hierarchy.</li>
+        <li>an optional <dfn>skosIndex</dfn>: An IRI containing an index of SKOS graphs.<br/>
+         These SKOS graphs can be used to describe the shape tree in human-readable terms.</li>
       </ul>
```

Comment:
(impacts the Shape Tree Structure section below.)

I think we want `skosIndex` on the ShapeTree doc itself rather than on the steps in the ShapeTree doc. Having them on the steps allows you to say, for ever step, "the interface hierarchies in this index will have arbitrary links back into steps in this ShapeTree and maybe others". Putting that assertion on the ShapeTree doc would be written
``` turtle
<> skosIndex <./MyKnownInterfaceHierarchies> .
```
and would make that statement about the ShapeTree document instead of each step. I think the latter is consistent with the #describing-shapetrees section you added below.

Reply:
I moved this location both in the spec and primer.  Especially on the spec side, not sure if I captured the description as crisply as I should.

Reply:
Here's what I'm seeing now:

> Informing all steps is an optional skosIndex: An IRI containing an index of SKOS graphs.
> the interface hierarchies in this index will have arbitrary links back into steps in this ShapeTree and maybe others.

This isn't really explaining to me what this skosIndex is for, and since we're in a primer document we don't need to be as terse. A few things that might help:
- Link to the Describing Shape Trees section and explain that this Index of SKOS graphs is meant to provide an extensible means in which the shape tree can be described using SKOS. 
- Explain that the skosIndex allows maintainers to provide a pointer in the document to SKOS graphs that accurately and reliably describe the shape tree as they intended.

This sentence in particular is pretty hard to understand:
> the interface hierarchies in this index will have arbitrary links back into steps in this ShapeTree and maybe others.

Reply:
This should be clearer now.